components/MetadataEditorDialog.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <v-dialog
+    content-class="metadata-dialog"
+    :value="dialog"
+    max-width="70%"
+    @click:outside="$emit('update:dialog', false)"
+  >
+    <metadata-editor
+      :item-id="itemId"
+      :force-refresh.sync="forceRefresh"
+      @cancel="close"
+      @save="close"
+    ></metadata-editor>
+  </v-dialog>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+export default Vue.extend({
+  props: {
+    dialog: {
+      type: Boolean,
+      required: true,
+      default: false
+    },
+    itemId: {
+      type: String,
+      default: ''
+    }
+  },
+  data() {
+    return {
+      forceRefresh: false
+    };
+  },
```

Comment:
```suggestion
  },
  computed: {
    calcMaxWidth() {
      return window.innerWidth < 700 ? '100%' : '60%';
    }
  },
```
This makes the metadata slightly more friendly on smaller screens.

Reply:
We should use the Vuetify breakpoints for this.

See https://vuetifyjs.com/en/features/breakpoints/#breakpoint-service-object

Reply:
Also for mobile, the dialog should likely be full screen, with the tabs moved to an horizontal tab bar at the top.

Reply:
Yep that is a better method